Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -90,6 +91,17 @@ WarpXParticleContainer::WarpXParticleContainer (AmrCore* amr_core, int ispecies)
     SetParticleSize();
     ReadParameters();
 
+    // This needs to be here since ReadParameters is static but the m_E_external_particle and B are not
+    // Should they be declared static?
```

Comment:
This is ok, not ideal that we use so many statics that `ReadParameters` is static, too.
```suggestion
```